src/main/java/TestRailIntegration/TestRailIntegration.java

```diff
@@ -0,0 +1,160 @@
+package TestRailIntegration;
+
+import com.codepine.api.testrail.TestRail;
+import com.codepine.api.testrail.model.Project;
+import com.codepine.api.testrail.model.Result;
+import com.codepine.api.testrail.model.Run;
+import com.codepine.api.testrail.model.Suite;
+
+import java.text.SimpleDateFormat;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class TestRailIntegration {
+
+    private static TestRail testRail;
+    protected final static String endPoint = "https://kovaloleksandra.testrail.io/";
+    protected final static String username = "oleksandra_koval@epam.com";
```

Comment:
these are candidates for env properties. Please create with the appropriate reader